test/integration/finance/invoice_download_test.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Finance::InvoiceDownloadTest < ActionDispatch::IntegrationTest
+  setup do
+    @invoice = FactoryBot.create(:invoice, state: :finalized)
+  end
+
+  teardown do
+    @invoice.pdf.clear
+    @invoice.save
```

Comment:
It is removing the invoice pdf file. It will stay on disk after testing. Next test will create an invoice with another id and probably another account. Not overwrite this one.

Reply:
Makes sense 👍